cx_Freeze/macdist.py

```diff
@@ -237,6 +237,12 @@ def create_plist(self):
         # Ensure CFBundleExecutable is set correctly
         contents["CFBundleExecutable"] = self.bundle_executable
 
+        # Specify that bundle is an application bundle
+        contents["CFBundlePackageType"] = "APPL"
+
+        # Cause application to run in high-resolution mode by default
+        contents["NSHighResolutionCapable"] = "True"
```

Comment:
Is this value set to true but the user can change it in the plist_items?
Can you document this?

Reply:
Done.  I also tweaked the code so that the new default values will not be used if there is a custom_info_plist specified.